components/Tooltip.js

```diff
@@ -22,23 +23,36 @@ class Tooltip extends Component {
     showIcon: true,
     iconName: 'info--glyph',
     iconDescription: 'tooltip',
+    positionThreshold: 5,
```

Comment:
@Noodles006 apologies was out last week.

For clarification - why 5 / why a prop at all? Seems to me the default behavior should be that if it exceeds a certain distance away to always recalculate the position